substrate/frame/nomination-pools/benchmarking/src/lib.rs

```diff
@@ -844,6 +883,146 @@ frame_benchmarking::benchmarks! {
 		assert!(&Pools::<T>::check_ed_imbalance().is_ok());
 	}
 
+	apply_slash {
```

Comment:
@georgepisaltu I am not super happy with the benchmarks since some code paths are never reached with `TransferStake` as adapter. Let me know if you have some better ideas to manage this (or possibly refactor adapter setup in the pallet).